reposerver/repository/repository.go

```diff
@@ -2263,6 +2263,44 @@ func (s *Service) GetRevisionMetadata(ctx context.Context, q *apiclient.RepoServ
 	return metadata, nil
 }
 
+func (s *Service) GetRevisionChartDetails(ctx context.Context, q *apiclient.RepoServerRevisionChartDetailsRequest) (*v1alpha1.ChartDetails, error) {
+	details, err := s.cache.GetRevisionChartDetails(q.Repo.Repo, q.Name, q.Revision)
+	if err == nil {
+		log.Infof("revision chart details cache hit: %s/%s/%s", q.Repo.Repo, q.Name, q.Revision)
+		return details, nil
+	} else {
+		if err != reposervercache.ErrCacheMiss {
+			log.Warnf("revision metadata cache error %s/%s/%s: %v", q.Repo.Repo, q.Name, q.Revision, err)
+		} else {
+			log.Infof("revision metadata cache miss: %s/%s/%s", q.Repo.Repo, q.Name, q.Revision)
```

Comment:
no, we log it on 2273 when it is something unexpected
on 2275 we know it's a cache miss, this is expected 
I will just switch the order to avoid the negative condition